### PR TITLE
Cleanup Scenery Window for second scroll area

### PR DESCRIPTION
**Pull request by duncanspumpkin**
I had a little attempt at pulling off scrollable tabs and the attached refactors will be needed so that there aren't issues.
I think after that I will change the window to a class before attempting to do the scrollable tabs again.

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
Remember to fix the code formatting so the other CIs can kick in

File: src/openrct2-ui/windows/Scenery.cpp [hunk 1]
```diff
@@ -836,14 +838,22 @@ static void WindowSceneryUpdate(rct_window* w)
     }
 }
 
+static void WindowSceneryContentScrollGetSize(rct_window* w, int32_t* height)
+{
+    auto rows = WindowSceneryCountRows(w);
+    *height = WindowSceneryRowsHeight(rows);
+}
+
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Scenery.cpp:846]: Why making it an out parameter and not a return value? Sounds more complex than it should be
```suggestion
static int32_t WindowSceneryContentScrollGetSize(rct_window* w)
{
    auto rows = WindowSceneryCountRows(w);
    return WindowSceneryRowsHeight(rows);
}

```

Comment (duncanspumpkin) [reply src/openrct2-ui/windows/Scenery.cpp:846]: that can be addressed when it is refactored into a class. As is for now it follows the style of all the other windows.

Comment (tupaschoal) [reply src/openrct2-ui/windows/Scenery.cpp:846]: Ok

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**